example/ios/Podfile.lock

```diff
@@ -6,14 +6,14 @@ PODS:
   - boost (1.76.0)
   - CocoaAsyncSocket (7.6.5)
   - DoubleConversion (1.1.6)
-  - FBLazyVector (0.69.9)
-  - FBReactNativeSpec (0.69.9):
+  - FBLazyVector (0.69.0)
```

Comment:
Not sure it matters, but also not sure why these versions are decreasing?

Reply:
good catch thank you - they should not have been. The react-native version was updated to 0.69.0 but my local version was out of sync.